content/features/01-graph-health.adoc

```diff
@@ -7,4 +7,4 @@ subTitle: "Have a checkup with your mesh's health"
 weight: 01
 ---
 
-Colors in the graph represent the **health** of your service mesh. If something is red or orange in your service mesh, then it might need attention. The color of the lines between applications represents the health of the **requests** between applications. The color around the application circle represents the health of the **application** itself.
+Colors in the graph represent the **health** of your service mesh. If something is red or orange in your service mesh, then it might need attention. The **side panel** provides information about what is selected, if something is unhealthy click on it and check the side panel for the **reason**.
```

Comment:
I would replace "reason" with "root cause" or "more information"

Reply:
Ok, I will use 'more information'.

Root cause is not the right term. For instance, we might know a workload is unhealthy because it only has 3 running replicas when 4 have been requested. But we don't know the root cause of why that one replica is not running.